### PR TITLE
Add post endpoint for getting photo

### PR DESCRIPTION
**Pull request by TANKVENERA**
Add post endpoint

### CODE REVIEW

File: app/repository/meteorsRepository.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-const apiPath = 'neo/rest/v1/feed';
-
 const fetchMeteors = async (dateFrom, dateTo) => {
+    const baseUrl = config.apiUrl || 'https://api.nasa.gov';
+    const apiKey = config.apiKey || 'DEMO_KEY';
+    const apiPath = 'neo/rest/v1/feed';
+
     const apiUrl = new URL(path.join(apiPath), baseUrl);
     apiUrl.searchParams.append("api_key", apiKey);
     apiUrl.searchParams.append("start_date", dateFrom);
```
Comment (mdziarkach) [inline app/repository/meteorsRepository.js:12]: Alternative variant

```js
return axios.get(`${config.apiUrl}/neo/rest/v1/feed`, {
  params: {
    api_key: config.apiKey,
    start_date: dateFrom,
    end_date: dateTo
});
```

File: app/repository/photoRepository.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+const fetchPhotos = async () => {
+    const baseUrl = config.apiUrl || 'https://api.nasa.gov'
+    const apiKey = config.apiKey || 'DEMO_KEY'
+    const apiPath = 'mars-photos/api/v1/rovers/curiosity/latest_photos';
+
+    const apiUrl = new URL(path.join(apiPath), baseUrl);
+    apiUrl.searchParams.append("api_key", apiKey);
+
+    return await axios.get(apiUrl.toString());
```
Comment (mdziarkach) [inline app/repository/photoRepository.js:13]: same one
```js
return axios.get(`${config.apiUrl}/mars-photos/api/v1/rovers/curiosity/latest_photos`, {
  params: {
    api_key: config.apiKey
});

```

File: app/routes/photoRoute.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+router.post('/latestPhoto', bodyParser.json(), async (req, res, next) => {
+    try {
+        const userId = req.body.userId;
+        const userName = req.body.userName;
+        console.log('Validator to be added for:', userId, userName)
+
+        const latestPhoto = await useCase.getLatestPhoto();
+        res.json(latestPhoto)
+    } catch (err) {
+        next(new NasaProxyException(err.status, `Downstream error: ${err.message}`))
```
Comment (mdziarkach) [inline app/routes/photoRoute.js:23]: next(err)

File: app/repository/meteorsRepository.js (hunk 1 above)
Comment (TANKVENERA) [reply app/repository/meteorsRepository.js:12]: will fix, thx

File: app/repository/photoRepository.js (hunk 2 above)
Comment (TANKVENERA) [reply app/repository/photoRepository.js:13]: will fix , thx

File: app/routes/photoRoute.js (hunk 3 above)
Comment (TANKVENERA) [reply app/routes/photoRoute.js:23]: Didn't catch the idea ) Do you mean  I shouldn't create NasaProxyException ?  And  just do : next(err) instead ?

**Pull request merged**